coordinator/persistence-api/src/main/java/org/apache/cassandra/stargate/transport/ProtocolVersion.java

```diff
@@ -38,7 +38,9 @@ public enum ProtocolVersion implements Comparable<ProtocolVersion> {
   V2(2, "v2", false), // no longer supported
   V3(3, "v3", false),
   V4(4, "v4", false),
-  V5(5, "v5-beta", true);
+  V5(5, "v5-beta", true),
```

Comment:
I thought v5 wasn't in beta anymore

Reply:
I think it is beta in DSE still.  It is out of beta only C* 4.x.